docs/preview/features/message-pumps/service-bus-azure-functions.md

```diff
@@ -113,9 +98,9 @@ namespace MessageProcessing
     {
         public override void Configure(IFunctionsHostBuilder builder)
         {
-            builder.Services.AddServiceBusMessageRouting()
-                            .WithServiceBusMessageHandler<OrderServiceBusMessageHandler, Order>(messageContext => messageContext.UserProperties["MessageType"] == MessageType.Order)
-                            .WithServiceBusMessageHandler<ShipmentServiceBusMessageHandler, Shipment>(messageContext => messageContext.UserProperties["MessageType"] == MessageType.Shipment);
+            builder.AddServiceBusMessageRouting()
+                   .WithServiceBusMessageHandler<OrderServiceBusMessageHandler, Order>(messageContext => messageContext.UserProperties["MessageType"] == MessageType.Order)
+                   .WithServiceBusMessageHandler<ShipmentServiceBusMessageHandler, Shipment>(messageContext => messageContext.UserProperties["MessageType"] == MessageType.Shipment);
```

Comment:
Let's use OrderV1 and OrderV2 as examples

Reply:
done that.